### PR TITLE
Try to suggest clang format changes

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Add bot for code style clang format info

### Tickets:
 - NA

### CODE REVIEW

**Comment by ilyachur** [comment]
Changes from external PR don't have write access to the original OpenVINO repository:
Screenshot 2022-10-28 at 07 39 52

After the merge such changes will look like:
Screenshot 2022-10-28 at 07 41 22

**Comment by ilyachur** [comment]
https://github.com/ilyachur/openvino/pull/50

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**
